name.abuchen.portfolio.tests/src/name/abuchen/portfolio/datatransfer/pdf/erstebank/ErsteBankPDFExtractorTest.java

```diff
@@ -1982,8 +2070,8 @@ public void testDividende08()
 
         assertThat(transaction.getMonetaryAmount(),
                         is(Money.of(CurrencyUnit.EUR, Values.Amount.factorize(25.70))));
-        assertThat(transaction.getGrossValue(),
-                        is(Money.of(CurrencyUnit.EUR, Values.Amount.factorize(29.86))));
+//        assertThat(transaction.getGrossValue(),
+//                        is(Money.of(CurrencyUnit.EUR, Values.Amount.factorize(29.86))));
```

Comment:
What is the reason to comment out this code?

Reply:
I had tested something and forgot to activate it again... thanks!
Done...